js/templates/nav-data.hbs

```diff
@@ -65,8 +70,8 @@
         </div>
         <ul>
           <li class="mega__item"><a href="{{webAppUrl}}/filings">All filings</a></li>
-          <li class="mega__item is-disabled">Authorized committees</li>
-          <li class="mega__item is-disabled">Unauthorized committees</li>
+          <li class="mega__item is-disabled">Authorized committee reports</li>
+          <li class="mega__item is-disabled">Unauthorized committees reports</li>
```

Comment:
Is the copy intentionally use different pluralization? Authorized singular committee vs Unauthorized plural committees?